contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_step_2_computer.handlebars

```diff
@@ -0,0 +1,49 @@
+<div id="clipboard" class="pull-right">
+	<div class="clipboard_header">
+		<h3 class="text-center">
+			<img src="/static/img/add_folder.png" alt="add" class="pull-left"/>
+			Adding Content to Topic
+			<a class="back pull-right btn-link toggle_clipboard">X</a>
+		</h3>
+		<table>
+			<tr>
+				<td class="text-center">1<h6>Source</h6></td>
+				<td class="text-center"><canvas></canvas></td>
+				<td class="text-center"><h4 class="text-center">2</h4><h6>Choose Content</h6></td>
+				<td class="text-center"><canvas></canvas></td>
+				<td class="text-center">3<h6>Metadata</h6></td>
+			</tr>
+		</table>
+	</div>
+	<div class="clipboard_content">
+		<label>2. Adding content from My Computer</label>
```

Comment:
'2' seemingly unmatched by a preceding '1'. Using more semantic HTML (as suggested above) could help to reduce this kind of issue.